### DISCUSSION TITLE
Is it possible to edit minor things of precompiled 15x15 ubl w/ probe version?

### DISCUSSION THREAD

**Discussion by hkasap83** [Jyers/Marlin#471] [Q&A] [answered]
I see the  configuration templates but there is 10x10 ubl one and i don’t know when i compiled them if there will be missing things or they will be the exactly the same with the already precompiled ones.

I know my bed size and xy limits with my probe offsets to give the exact points to the probe. I want to measure 225 coordinates in means of a square or rectangle without changing the rest of the original precompiled firmware. 

Nozzle can go to X(245) so probe can touch 201 (with -44 probe offset)
Nozzle can go to Y(230), so probe can touch Y(224) (with -6 probe offset)

For example by using these x and y values;
![image](https://user-images.githubusercontent.com/80889293/114442419-338ab600-9bd5-11eb-8f3d-533cc5292054.png)

**Comment by Jyers** [comment]
I'm not entirely sure what you're asking. You can not edit a precompiled file, but if you want to compile your own you're welcome to. Just start with the UBL BL Touch 10x10 config templates as a base.

**Comment by hkasap83** [comment, marked as answer]
> I'm not entirely sure what you're asking. You can not edit a precompiled file, but if you want to compile your own you're welcome to. Just start with the UBL BL Touch 10x10 config templates as a base.

To summarize, I want to give 200 or 225 exact coordinates to probe the bed to eliminate the computed ones and the rest of your firmware will remain as it is.  Maybe your next release will enable us to enter or edit these points on the screen.

I understand that the only way for now is compiling own from the templates.

**Reply by Jyers** [reply]
There is no way I can make such settings modifiable from the screen due to marlin's limitations. I can assure you this is not really necessary, the inferred points are only on the perimeter and therefore are fairly accurate plus if you really want to you can set them manually from the UBL menu. Either way if you still feel you want this you will need to compile your own.

**Reply by G4GNX** [reply]
If I'm right, UBL does not probe a square pattern, so you will never get it to probe the way I think you want.
I had issues with UBL and haven't had time to investigate, so I used the bilinear template and in my case, just made it 10x10.
15x15 take over 40 minutes, so I thought I'd try 10x10 to see if it had enough accuracy. So far so good......... :-D

**Reply by Jyers** [reply]
^^^

This is also why you don't really want ubl to be able to reach all the points... 40 minutes is not ideal.

**Reply by hkasap83** [reply]
Thank you all for your attention, I understand the main approach.
